Lib/abc.py

```diff
@@ -28,7 +28,13 @@ def my_abstract_method(self, ...):
 class abstractclassmethod(classmethod):
     """A decorator indicating abstract classmethods.
 
-    Deprecated, use 'classmethod' with 'abstractmethod' instead.
+    Deprecated, use 'classmethod' with 'abstractmethod' instead:
+
+        @classmethod
+        @abstractmethod
+        def my_abstract_classmethod(cls, ...):
+            ...
```

Comment:
Could this indented method definition be confusing for some without more context?

Perhaps add one more line for context, `class C(ABC):`, as in the docs?
```suggestion
class C(ABC):
        @classmethod
        @abstractmethod
        def my_abstract_classmethod(cls, ...):
            ...
```